lib/engine/step/waterfall_auction.rb

```diff
@@ -66,14 +68,14 @@ def actions(entity)
 
       def setup
         setup_auction
-        @companies = @game.companies.sort
+        @companies = @game.companies.sort_by(&:value)
```

Comment:
is this correct for all games? seems like company line 31 sorts by min_bid, name

Reply:
Micheal recently added the <=> operator to company for 1846 2P, so no impact to the waterfall auction games. In my opinion, the company class shouldn't dictate sorting for auctions. Each auction type is different with different needs.

Looking at his commit. I did remove a sort that he added in the view. I will re-add that.

Reply:
I added the sorts to the draft auction and verified they work correctly.